snippets.go

```diff
@@ -281,3 +282,33 @@ func (s *SnippetsService) ExploreSnippets(opt *ExploreSnippetsOptions, options .
 
 	return ps, resp, nil
 }
+
+// ListAllSnippetsOptions represents the available ListAllSnippets() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/snippets.html#list-all-snippets
+type ListAllSnippetsOptions struct {
+	ListOptions
+	CreatedAfter      *time.Time `url:"created_after,omitempty" json:"created_after,omitempty"`
+	CreatedBefore     *time.Time `url:"created_before,omitempty" json:"created_before,omitempty"`
```

Comment:
These fields should use `*ISOTime` instead

Reply:
Update, thanks.